frontend/src/data/api-service.ts

```diff
@@ -97,14 +94,16 @@ export const createApiService = ({
   accessToken = undefined,
   isVersioned = true,
 }: ApiServiceConfig = {}): ApiService => {
+  const headers: OpenverseAxiosRequestConfig["headers"] = isServer
+    ? { "User-Agent": userAgent }
+    : {}
+  if (accessToken) {
+    headers["Authorization"] = `Bearer ${accessToken}`
+  }
```

Comment:
```suggestion
  const headers: OpenverseAxiosRequestConfig["headers"] = {}
  
  if (isServer) {
  	headers["User-Agent"] = userAgent
  }
  
  if (accessToken) {
    headers["Authorization"] = `Bearer ${accessToken}`
  }
```

I think something like this is more consistent so we're assigning each of our optional headers in the same manner.